admin/database/database.go

```diff
@@ -179,9 +179,10 @@ type DB interface {
 	DeleteProjectInvite(ctx context.Context, id string) error
 	UpdateProjectInviteRole(ctx context.Context, id, roleID string) error
 
-	FindBookmarks(ctx context.Context, projectID, userID string) ([]*Bookmark, error)
+	FindBookmarks(ctx context.Context, projectID, dashboardName, userID string) ([]*Bookmark, error)
```

Comment:
Maybe now that we have the resource kind/name terminology, we should generalize this to `resourceKind, resourceName string`?